test/Microsoft.ML.Tests/Scenarios/Api/TestApi.cs

```diff
@@ -20,7 +20,7 @@ namespace Microsoft.ML.Tests.Scenarios.Api
 {
     public partial class TestApi : BaseTestClass
     {
-        private const string OutputRelativePath = @"..\Common\Api";
+        private const string OutputRelativePath = @"../Common/Api";
```

Comment:
Note: Typically it is best to use `Path.Combine` to build up paths when you want the code to work cross-platform.

```C#
private const string OutputRelativePath = Path.Combine("..", "Common", "Api");
```